Utilities/DataFlow/src/SubframeBuilderDevice.cxx

```diff
@@ -13,6 +13,10 @@
 #include "Headers/DataHeader.h"
 #include "FairMQProgOptions.h"
 
+// temporary hack to solve issue 279; linking problem for this constexpr
+// variable
+constexpr uint32_t AliceO2::DataFlow::SubframeBuilderDevice::mDuration;
+
```

Comment:
do you think this still has the correct initialization? As the initialization is in the header file.